web/src/components/tables/sample-table.tsx

```diff
@@ -44,6 +45,12 @@ export const SampleTable: FC<Props> = ({ data, readOnly = false, onChange, heigh
           height={height || tableHeight}
           readOnly={readOnly}
           colHeaders={true}
+          colWidths={200}
+          rowHeights={23}
+          renderer={(instance, td, row, col, prop, value, cellProperties) => {
```

Comment:
Cool. This looks minimal changes, so if it works for you locally I trust to deploy it

Reply:
It works, but also can be improved in the future